README.CentOS7.md

```diff
@@ -76,6 +76,12 @@ sudo yum install nodejs
 sudo curl -sS https://getcomposer.org/installer | php
 sudo mv composer.phar /usr/local/bin/composer
 ```
+### Extensions for Composer
+```bash
+sudo yum install php-xml
+sudo yum install php-mbstring
```

Comment:
These package are already installed at line 37.

Ensure that php will use version 7.2 instead rather that default version.

Reply:
@PapillonMcGill hmm I didn't notice php72-php-mbstring and php72-php-xml That's weird because I received the error output after the https://github.com/aces/Loris/pull/5489 fix. I'm kind of stumped on why. I'll try re-testing on a fresh install of CentOS and see what happens.

Reply:
@PapillonMcGill, I just re-tested and I'm still getting the output: 
```
Problem 1
    - Installation request for phar-io/manifest 1.0.1 -> satisfiable by phar-io/manifest[1.0.1].
    - phar-io/manifest 1.0.1 requires ext-dom * -> the requested PHP extension dom is missing from your system.
  Problem 2
    - Installation request for phpmd/phpmd 2.7.0 -> satisfiable by phpmd/phpmd[2.7.0].
    - phpmd/phpmd 2.7.0 requires ext-xml * -> the requested PHP extension xml is missing from your system.
  Problem 3
    - Installation request for phpunit/php-code-coverage 5.3.2 -> satisfiable by phpunit/php-code-coverage[5.3.2].
    - phpunit/php-code-coverage 5.3.2 requires ext-dom * -> the requested PHP extension dom is missing from your system.
  Problem 4
    - Installation request for phpunit/phpunit 6.5.14 -> satisfiable by phpunit/phpunit[6.5.14].
    - phpunit/phpunit 6.5.14 requires ext-dom * -> the requested PHP extension dom is missing from your system.
  Problem 5
    - Installation request for squizlabs/php_codesniffer 3.5.1 -> satisfiable by squizlabs/php_codesniffer[3.5.1].
    - squizlabs/php_codesniffer 3.5.1 requires ext-simplexml * -> the requested PHP extension simplexml is missing from your system.
  Problem 6
    - Installation request for theseer/tokenizer 1.1.3 -> satisfiable by theseer/tokenizer[1.1.3].
    - theseer/tokenizer 1.1.3 requires ext-dom * -> the requested PHP extension dom is missing from your system.

  To enable extensions, verify that they are enabled in your .ini files:
    - /etc/php.ini
    - /etc/php.d/20-bz2.ini
    - /etc/php.d/20-calendar.ini
    - /etc/php.d/20-ctype.ini
    - /etc/php.d/20-curl.ini
    - /etc/php.d/20-exif.ini
    - /etc/php.d/20-fileinfo.ini
    - /etc/php.d/20-ftp.ini
    - /etc/php.d/20-gettext.ini
    - /etc/php.d/20-iconv.ini
    - /etc/php.d/20-json.ini
    - /etc/php.d/20-phar.ini
    - /etc/php.d/20-sockets.ini
    - /etc/php.d/20-tokenizer.ini
  You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.

```
after running the `composer install` command.

Reply:
have you checked in these ini files to see if php7.2 extension are enabled?

Reply:
@PapillonMcGill all of the extensions are enabled. Seems like the php72-php-mbstring and php72-php-xml are not placing .ini files inside /etc/php.d. Although, the new command does place extensions inside the directory with them enabled.